activities.el

```diff
@@ -648,8 +648,9 @@ activity's name is NAME."
     ;; HACK: Since `bookmark--jump-via' insists on calling a buffer-display
     ;; function after handling the bookmark, we use an immediate timer to
     ;; set the window configuration.
-    (run-at-time nil nil (lambda ()
-                           (window-state-put state (frame-root-window) 'safe)))))
+    (let ((frame (selected-frame)))
```

Comment:
Thanks, this looks like the right fix.  What if we bind that variable in the `pcase-let*` form above?

Reply:
Done.  In fact just a simple `let` will do.